chrome/fed-modules.json

```diff
@@ -685,5 +685,17 @@
                 ]
             }
         ]
+    },
+    "iam": {
+      "manifestLocation": "/apps/iam/fed-mods.json",
+      "modules": [
+          {
+              "id": "authentication-policy",
+              "module": "./RootApp",
+              "routes": [
+                  "/iam/authpolicy"
```

Comment:
@ryelo do we need some special setup in Akamai to allow a new bundle?

Reply:
No need to, it's just missing a main.yml and navigation update.